toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1001 @@
+    </message>
+</context>
+<context>
+    <name>PumpTool</name>
+    <message>
+        <source>Size:</source>
+        <translation>Größe:</translation>
+    </message>
+    <message>
+        <source>Accuracy:</source>
+        <translation>Genauigkeit:</translation>
+    </message>
+</context>
+<context>
+    <name>QObject</name>
+    <message>
+        <source>Min:</source>
+        <translation>Minimum:</translation>
+    </message>
+    <message>
+        <source>Max:</source>
+        <translation>Maximum</translation>
+    </message>
+    <message>
+        <source>Yes</source>
+        <translation>Ja</translation>
+    </message>
+    <message>
+        <source>No</source>
+        <translation>Nein</translation>
+    </message>
+    <message>
+        <source>The copied selection cannot be pasted in the current drawing.</source>
+        <translation>Die kopierte Auswahl kann nicht in die aktuelle Zeichnung eingefügt werden.</translation>
+    </message>
+    <message>
+        <source>The current column is locked.</source>
+        <translation>Die aktuelle Spalte ist gesperrt.</translation>
+    </message>
+    <message>
+        <source>The current column is not visible in Camera Stand.</source>
+        <translation type="vanished">Die aktuelle Spalte ist nicht sichtbar im Kamera Stand</translation>
+    </message>
+    <message>
+        <source>It is not possible to edit the audio column.</source>
+        <translation>Es ist nicht möglich die Audio-Spalte zu bearbeiten.</translation>
+    </message>
+    <message>
+        <source>It is not possible to edit the Magpie column.</source>
+        <translation>Es ist nicht möglich die Magpie-Spalte zu bearbeiten</translation>
+    </message>
+    <message>
+        <source>The current tool cannot be used on a Level column.</source>
+        <translation>Das aktuelle Werkzeug kann nicht mit einer Level-Spalte genutzt werden.</translation>
+    </message>
+    <message>
+        <source>The current tool cannot be used on a Mesh column.</source>
+        <translation>Das aktuelle Werkzeug kann nicht mit einer Mesh-Spalte genutzt werden.</translation>
+    </message>
+    <message>
+        <source>The current tool cannot be used in Level Strip mode.</source>
+        <translation>Das aktuelle Werkzeug kann nicht im Level Strip Modus genutzt werden.</translation>
+    </message>
+    <message>
+        <source>The current tool cannot be used to edit a motion path.</source>
+        <translation>Das aktuelle Werkzeug kann nicht zum bearbeiten eines Bewegungspfad genutzt werden.</translation>
+    </message>
+    <message>
+        <source>The current level is not editable.</source>
+        <translation>Die aktuelle Ebene ist nicht bearbeitbar.</translation>
+    </message>
+    <message>
+        <source>The current tool cannot be used on a Vector Level.</source>
+        <translation>Das aktuelle Werkzeug kann nicht auf einer Vector-Ebene genutzt werden.</translation>
```

Comment:
Should probably be "kann nicht auf [...] angewendet werden." (see also variants below).